blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.57-jm-c",
+  "version": "0.0.57-jm-d",
```

Comment:
Published but won't turn up if you ask for "latest". But the version in the branch is the version the branch depends on :)